ros2_generate_interface_docs/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'ros2_generate_interface_docs'
+
+setup(
+    name=package_name,
+    version='0.0.1',
```

Comment:
Before the first release this should be 0.0.0 since 0.0.1 hasn't been released yet.